src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Service/Gateway/RespondService.php

```diff
@@ -48,29 +49,25 @@ class RespondService
     /** @var SecondFactorTypeService */
     private $secondFactorTypeService;
 
-    /**
-     * SecondFactorRespondService constructor.
-     * @param SamlAuthenticationLogger $samlLogger
-     * @param LoaResolutionService $loaResolutionService
-     * @param LoaAliasLookupService $loaAliasLookupService
-     * @param ResponseFactory $responseFactory
-     * @param SecondFactorService $secondFactorService
-     * @param SecondFactorTypeService $secondFactorTypeService
-     */
+    /** @var ResponseValidator */
+    private $responseValidator;
+
     public function __construct(
         SamlAuthenticationLogger $samlLogger,
         LoaResolutionService $loaResolutionService,
```

Comment:
Candidate for constructor property promotion

Reply:
Yes, when we are upgrading to php8 we surely will do that. For now this code must be PHP72 compat.